ipam_test.go

```diff
@@ -534,6 +534,72 @@ func TestSpace(t *testing.T) {
 	}
 }
 
+func TestConains(t *testing.T) {
+	tests := []struct {
+		Network          string
+		Subnet           string
+		ExpectedContains bool
+	}{
+		{ // Test 0.
```

Comment:
I would like comments to have their own line.

Reply:
Done.